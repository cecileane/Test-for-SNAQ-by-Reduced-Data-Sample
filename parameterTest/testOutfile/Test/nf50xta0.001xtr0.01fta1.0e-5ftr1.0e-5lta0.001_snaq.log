optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=1.0e-5,
                       xtolAbs=0.001, xtolRel=0.01.
 max number of failed proposals = 50, liktolAbs = 0.001.
rootname for files: nf50xta0.001xtr0.01fta1.0e-5ftr1.0e-5lta0.001_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 70677
seed: 70677 for run 1
Sat Oct  1 17:41:37 2016

 BEGIN SNaQ for run 1, seed 70677 and hmax 1
Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 88 (failures: 50) with -loglik=158.61503 and ht_min=[0.31865,0.00549,10.0,0.32446,1.11438]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 27		 27		 3	 |	 0		 0		 --
mvtarget	 24		 24		 0	 |	 0		 0		 --
chdir		 13		 13		 1	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 23		 16		 1	 |	 4		 4		 --
Total		 88		 81		 6	 |	 4		 4		 2
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 0.5

 FINISHED SNaQ for run 1, -loglik of best 158.61502568730597
(3,(((2,#H7:::0.3186452789887333):10.0,1):1.1143843398701665,(5,6):0.32446435485541164):0.0054895002271946845,(4)#H7:::0.6813547210112667);
---------------------
seed: 75810 for run 2
Sat Oct  1 17:41:41 2016

 BEGIN SNaQ for run 2, seed 75810 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 65 (failures: 50) with -loglik=155.49626 and ht_min=[0.12573,0.33535,0.55437,0.32983]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 17		 17		 1	 |	 0		 0		 --
mvtarget	 25		 25		 1	 |	 0		 0		 --
chdir		 8		 8		 2	 |	 1		 1		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 14		 13		 0	 |	 10		 10		 --
Total		 65		 64		 5	 |	 11		 11		 5
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 0.5

 FINISHED SNaQ for run 2, -loglik of best 155.49625734552802
(((3,(6,5):0.33534741435606513):0.12573292165936892,(1,(2)#H7:::0.6269753002672888):2.155860102725774):2.155860102725774,4,#H7:::0.37302469973271124);
---------------------

Sat Oct  1 17:41:46 2016
MaxNet is (((3,(6,5):0.33534741435606513):0.12573292165936892,(1,(2)#H7:::0.6269753002672888):2.155860102725774):2.155860102725774,4,#H7:::0.37302469973271124); 
with -loglik 155.49625734552802
