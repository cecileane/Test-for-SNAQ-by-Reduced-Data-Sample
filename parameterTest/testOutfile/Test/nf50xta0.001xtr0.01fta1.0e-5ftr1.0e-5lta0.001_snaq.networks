(((3,(6,5):0.33534741435606513):0.12573292165936892,(1,(2)#H7:::0.6269753002672888):2.155860102725774):2.155860102725774,4,#H7:::0.37302469973271124);, with -loglik 155.49625734552802 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
(3,((4)#H7:::0.6636040991338321,(1,(2,#H7:::0.3363959008661679):1.879177457280744):1.1090222981293458):0.0,(6,5):0.32474779108246554);, with -loglik 161.74063317059813
(((3,(6,5):0.31725019954715117):0.0,#H7:::0.48137952164300535):0.566537308665028,4,(2,(1)#H7:::0.5186204783569947):6.534605182887163);, with -loglik 170.72930286565588
(#H7:1.0035002573035845::0.3816425472901903,4,(2,(((3,(6,5):0.3713571998689194):0.0)#H7:9.993879579132516::0.6183574527098097,1):0.39281142689061577):9.829394273674136);, with -loglik 181.42851198954412
