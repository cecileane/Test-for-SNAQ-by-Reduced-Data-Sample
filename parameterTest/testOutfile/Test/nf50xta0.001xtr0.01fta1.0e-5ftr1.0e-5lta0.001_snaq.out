(((3,(6,5):0.33534741435606513):0.12573292165936892,(1,(2)#H7:::0.6269753002672888):2.155860102725774):2.155860102725774,4,#H7:::0.37302469973271124); -Ploglik = 155.49625734552802
 Dendroscope: (((3,(6,5):0.33534741435606513):0.12573292165936892,(1,(2)#H7):2.155860102725774):2.155860102725774,4,#H7);
 Elapsed time: 9.288257805 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 (((3,(6,5):0.33534741435606513):0.12573292165936892,(1,(2)#H7:::0.6269753002672888):2.155860102725774):2.155860102725774,4,#H7:::0.37302469973271124);, with -loglik 155.49625734552802
 (3,(((2,#H7:::0.3186452789887333):10.0,1):1.1143843398701665,(5,6):0.32446435485541164):0.0054895002271946845,(4)#H7:::0.6813547210112667);, with -loglik 158.61502568730597
-------
